### ISSUE TITLE
How to  apply a model from Python

### ISSUE THREAD

**Issue by haesleinhuepf** [bioimage-io/core-bioimage-io-python#391]
Hi all,

I would like to apply a bioimage-io model to an image using Python. Unfortnunately, I cannot find the documentation that shows how to do this. Furthermore, the chatbot haluzinates API calls that don't work. Can anyone point me to the right place in the documentation, ideally a reproducible Jupyter notebook?

Thanks!

This is what the chatbot told me (2 attempts):

![image](https://github.com/bioimage-io/bioimage.io/assets/12660498/14b9b4f7-6191-4e09-85a3-23a3df2e50c2)

**Comment by oeway** [comment]
Hi @haesleinhuepf Sorry to hear that!

We are in the process of updating the core library, and the api has changed and the documentation is lagging behind.


Previously, we have this one: https://github.com/bioimage-io/core-bioimage-io-python/blob/main/example/model_usage.ipynb

For the chatbot, since the documentation is not loaded/updated into the knowledge base, I won't be supprised that it provide you wrong anwser.

@FynnBe Could you comment what's the best way for @haesleinhuepf to move forward in using the updated api?
Would it work if he pin the core library to an older version and use the notebook we made previously?

**Comment by constantinpape** [comment]
Just to add: it would be ideal if we get back the functional API we had in the previous version of the `bioimageio.core` library. (With some adaptations if necessary.)

**Comment by FynnBe** [comment]
@haesleinhuepf thank you for reporting this issue.
I will add `load_model` and `load_resource` as aliases of `load_description` to match the previous API more closely

**Comment by haesleinhuepf** [comment]
Thanks everyone for the feedback! Just a note: having a stable API and high-quality documentation is key for establishing a technology/ method in Python. It is impossible otherwise to teach people how to use bio-image model zoo. Thanks again!

**Comment by constantinpape** [comment]
> Wow, this issue doesn't seem completed to me. Anyway, thanks for the feedback

No, this one is not closed. I think  #393 only addresses a narrow part of this by exposing the `load_model` function again.
The documentation for applying a model is still lacking.
